www/src/components/package-readme.js

```diff
@@ -39,8 +39,13 @@ const PackageReadMe = props => {
             },
           }}
           href={githubUrl}
+          aria-label={`Github source code for ${packageName}`}
```

Comment:
```suggestion
```
you only need one of them :)

Reply:
Thank you @wardpeet title attribute shows a tooltip when hovered. Not sure if it is useful for the user :)